pkg/policy/l3.go

```diff
@@ -74,10 +74,16 @@ func (m *CIDRPolicyMap) Insert(cidr string, ruleLabels labels.LabelArray) int {
 	if _, found := m.Map[key]; !found {
 		m.Map[key] = &CIDRPolicyMapRule{Prefix: *ipnet, DerivedFromRules: labels.LabelArrayList{ruleLabels}}
 		if ipnet.IP.To4() == nil {
-			m.IPv6Count++
+			m.IPv6PrefixCount[ones]++
 		} else {
-			m.IPv4Count++
+			m.IPv4PrefixCount[ones]++
+		}
+
+		if len(m.IPv6PrefixCount) > api.MaxCIDRPrefixLengths {
+			log.Infof("Inserting CIDR with prefix length %d/%d, generation may fail",
```

Comment:
Info level seems very noisy a priori. Change to Debug?

Reply:
Can't you return an error and add expose that through the endpoint's log?

Reply:
Not from here, this is during CIDRPolicyMap construction which is divorced from endpoints. I could signal it when we attempt to apply the policy to the endpoint and generate the BPF.

Reply:
Please file a GH to clean this up. It's not ideal but I don't want to hold up this PR.

Reply:
I removed this log.